### PR TITLE
Python 3 compatibility

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: osg-ca-certs-updater.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/python
 """OSG Auto-Updater for CA Certificates"""
 # (ignore bad name of script) pylint: disable=C0103
+from __future__ import absolute_import
```
Comment (edquist) [inline osg-ca-certs-updater.py]: Just curious, which module(s) use `absolute_import` ?

Comment (matyasselmeci) [reply osg-ca-certs-updater.py]: Ugh. None of them. python-modernize adds that import as a matter of course.

Comment (edquist) [reply osg-ca-certs-updater.py]: Wow, yeah, it looks so modern now.  Absolutely from the future.

**Review by edquist — APPROVED** [review]
Go Go Go!

**Pull request merged**